### ISSUE TITLE
Logs manager 

### ISSUE THREAD

**Issue by SamuelRocha2015** [SamuelRocha2015/travel-manager-api#7]
- Store and manager logs.
- Create statistics. f.ex: _"top ten places visited"_
- Technologies : Cassandra or Elastic logstash

**Comment by QueirozDavi** [comment]
I agree with that, this feature will add a lot of value for api, but just a question: why don't use redis instead of cassandra or elastic logstash? I'm thinking in a job running 1 time per day and updating redis cache database with those informations

**Comment by SamuelRocha2015** [comment]
Redis is a cache database, no a LOG database.
For the summary / statistics we can use redis but for the LOG storage we need a specific database.

**Comment by QueirozDavi** [comment]
Ok, now i understood your point.
